actions/pack_action.go

```diff
@@ -32,9 +32,10 @@ import (
 )
 
 var tarOpts = map[string]string{
-	"gz":    "z",
-	"bzip2": "j",
-	"xz":    "J",
+	"gz":    "gzip",
+	"bzip2": "bzip2",
+	"xz":    "xz",
+	"zstd":  "zstd",
```

Comment:
any reason to not make the options be e.g. `--gzip` to be more in line with the unpackTarOpts in archive?